lib/Mojolicious/Command/Author/generate/dockerfile.pm

```diff
@@ -0,0 +1,87 @@
+package Mojolicious::Command::Author::generate::dockerfile;
+use Mojo::Base 'Mojolicious::Command';
+
+use Mojo::File qw(path);
+use Mojo::JSON qw(encode_json);
+
+has description => 'Generate "Dockerfile"';
+has usage       => sub { shift->extract_usage };
+
+sub run {
+  my $self = shift;
+  my $name = $self->app->moniker;
+  my $exe  = $ENV{MOJO_EXE} ? path($ENV{MOJO_EXE})->to_rel($self->app->home)->to_string : "script/$name";
+  $self->render_to_rel_file('dockerfile', 'Dockerfile', {name => $name, cmd => encode_json [$exe, 'prefork']});
+}
+
+1;
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Command::Author::generate::dockerfile - Dockerfile generator command
+
+=head1 SYNOPSIS
+
+  Usage: APPLICATION generate dockerfile [OPTIONS]
+
+    mojo generate dockerfile
```

Comment:
This should say:
```
    ./myapp.pl generate dockerfile
    mojo generate dockerfile
```

Reply:
I've adjusted this as suggested but changed `mojo generate dockerfile` to `./script/my_app generate dockerfile` (to encourage folks to generate it from their app so it can be more correct), but happy to revert it (and the one in `Mojolicious::Commands`) to `mojo generate dockerfile` if you prefer. :+1: